crates/notary/server/src/service/axum_websocket.rs

```diff
@@ -584,17 +617,133 @@ impl Sink<Message> for WebSocket {
     }
 }
 
+/// UTF-8 wrapper for [Bytes].
+///
+/// An [Utf8Bytes] is always guaranteed to contain valid UTF-8.
```

Comment:
```suggestion
/// An [Utf8Bytes] is always guaranteed to contain valid UTF-8.
/// The following NOTARY_MODIFICATION(s) are required because `async_tungstenite` (v0.28.2) is using an older version of `tungstenite` than `tokio_tungstenite` (v0.26.1). This older version of `tungstenite` (v0.26.0) doesn't have `Utf8Bytes`.
```